lib/jira/commands/assign.rb

```diff
@@ -3,7 +3,7 @@ class CLI < Thor
 
     desc "assign", "Assign a ticket to a user"
```

Comment:
All of these descriptions should be changed to something like:

    desc "assign <TICKET>", "..."

To indicate the required param. Currently, for example, if you run `jira assign` without specifying a ticket name then you get a slightly odd error message:

    ERROR: "jira assign" was called with no arguments
    Usage: "jira assign"